Workbooks/Windows Virtual Desktop/shared/Host Pool Overview/Host Pool Overview.workbook

```diff
@@ -180,11 +180,26 @@
             },
             "timeContextFromParameter": "TimeRange",
             "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": null
+          },
+          {
+            "id": "c9f78080-1f60-4900-b137-3b9c319bde41",
+            "version": "KqlParameterItem/1.0",
+            "name": "HostpoolType",
+            "type": 1,
+            "query": "resources\r\n| where type == \"microsoft.desktopvirtualization/hostpools\" and resourceGroup == tolower(\"{ResourceGroup}\")  and id == \"{HostPool}\"\r\n| project hostPoolType= properties.hostPoolType",
+            "crossComponentResources": [
```

Comment:
Does ARG query language support the `=~` operator for case insensitive comparison?

Reply:
Updated to use '=~' operation for case-insensitive comparison.

Reply:
But then you don't need the `tolower()` function anymore